src/domain/event/queryUtils.ts

```diff
@@ -72,46 +72,67 @@ export const useSimilarEventsQuery = (
   return { data, loading };
 };
 
-const useOtherEventTimesVariables = (
-  event: EventFields,
-  eventType: EventTypeId = EventTypeId.General
-) => {
+const useOtherEventTimesVariables = (event: EventFields) => {
   const superEventId = React.useMemo(
     () => getEventIdFromUrl(event.superEvent?.internalId || '', 'event'),
     [event.superEvent]
   );
 
   const variables = React.useMemo(
     (): EventListQueryVariables => ({
-      include: ['keywords', 'location'],
+      // include: ['keywords', 'location'],
       sort: EVENT_SORT_OPTIONS.START_TIME,
       start: 'now',
       superEvent: superEventId,
-      eventType,
+      /* 
+      Since LE v2 is listing general type of events 
+      when no event type is given as a parameter,
+      this needs a list of eventTypes
+      or otherwise some events or courses are always excluded.
+      */
+      eventType: [EventTypeId.General, EventTypeId.Course] //event.typeId
     }),
-    [eventType, superEventId]
+    [superEventId]
   );
 
   return { superEventId, variables };
 };
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const useOtherEventTimes = (
+export const useSubEventsQueryVariables = (
```

Comment:
could be simplified:

```
export const useSubEventsQueryVariables = (
  event: EventFields
): { superEventId: string | undefined; variables: EventListQueryVariables } => {
  const variables = {
    sort: EVENT_SORT_OPTIONS.START_TIME,
    start: 'now',
    superEvent: event.id,
    /* 
      Since LE v2 is listing general type of events 
      when no event type is given as a parameter,
      this needs a list of eventTypes
      or otherwise some events or courses are always excluded.
      */
    eventType: [EventTypeId.General, EventTypeId.Course], //event.typeId
  };

  return { superEventId: event.id, variables };
};
```